service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/C21OrderEmailContentProviderTest.java

```diff
@@ -38,71 +39,102 @@
 
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration(classes = {JacksonAutoConfiguration.class, C21OrderEmailContentProvider.class,
-    HearingBookingService.class, LocalAuthorityNameLookupConfiguration.class, DateFormatterService.class})
+    HearingBookingService.class, LocalAuthorityNameLookupConfiguration.class, DateFormatterService.class,
+    CafcassLookupConfiguration.class})
 class C21OrderEmailContentProviderTest {
-    private final LocalDate date = LocalDate.now();
+    private final LocalDate today = LocalDate.now();
     private final DateFormatterService dateFormatterService = new DateFormatterService();
     private final HearingBookingService hearingBookingService = new HearingBookingService();
 
     private static final String LOCAL_AUTHORITY_CODE = "example";
+    private static final String CAFCASS_EMAIL = "FamilyPublicLaw+cafcass@gmail.com";
+    private static final String CAFCASS_NAME = "Example Cafcass";
 
-    @Autowired
-    private ObjectMapper objectMapper;
+    @MockBean
+    private CafcassLookupConfiguration cafcassLookupConfiguration;
 
     @MockBean
     private LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
 
+    @Autowired
+    private ObjectMapper objectMapper;
+
     private C21OrderEmailContentProvider c21OrderEmailContentProvider;
 
     private String familyManCaseNumber;
+    private String documentId;
+    private String subjectLine;
 
     @BeforeEach
     void setup() {
         this.c21OrderEmailContentProvider = new C21OrderEmailContentProvider("",
-            objectMapper, hearingBookingService, localAuthorityNameLookupConfiguration, dateFormatterService);
+            objectMapper, hearingBookingService, localAuthorityNameLookupConfiguration, dateFormatterService,
+            cafcassLookupConfiguration);
 
         given(localAuthorityNameLookupConfiguration.getLocalAuthorityName(LOCAL_AUTHORITY_CODE))
             .willReturn("Example Local Authority");
 
+        given(cafcassLookupConfiguration.getCafcass(LOCAL_AUTHORITY_CODE))
+            .willReturn((new CafcassLookupConfiguration.Cafcass(CAFCASS_NAME, CAFCASS_EMAIL)));
+
         familyManCaseNumber = RandomStringUtils.randomAlphabetic(8);
+        documentId = randomUUID().toString();
```

Comment:
Really picky, but if we have got documentId I believe it should be in UUID format and defer toString when it's required.